rpa-manager/modules/ROOT/pages/processoperations-deploymentmap.adoc

```diff
@@ -18,18 +26,58 @@ The interrelationships of processes, RPA Bots and Configurations are mapped by a
 |View the process deployment map
 |Process Deployment Map Open
 
+|Execute, pause or stop configurations
+|Process Automation Open
+
 |===
 
 * You can only view processes for which you are part of the process team.
-* To execute, pause or stop configurations obtain the needed team member permissions for the production phase.
+* To execute, pause or stop configurations obtain the needed team member permissions for the respective phase.
+* You cannot interact with invokable configurations.
+* The deployment map shows only bots that have configurations deployed.
+
+== Filter Configurations by Status
+
+To show a relevant subset of the map, filter the configurations by status.
```

Comment:
```suggestion
To show a subset of the map, filter the configurations by status.
```

Reply:
relevant is unnecessary, as the concept of filtering is understood by the audience